client/src/components/Notifications/NotificationDropDown.js

```diff
@@ -230,9 +270,24 @@ const menu = (notifications, organisationId, t) => {
         </Link>
       </Menu.Item>
       <div className="notifications-container">
+        {notifications.length > 0 && 
+          <ClearAllNotifications>
+            <ClearAllButton
+              onClick={() => {
+                notifications.forEach((notification) => {
+                  clearNotification(notification._id);
```

Comment:
Yeah this definitely needs to be packed into one single event, so all of this can be a single query.
People might have a lot of notifications, and this is not ideal.